debug.log

```diff
@@ -0,0 +1,10 @@
+[0928/032233.493:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
```

Comment:
we don't want this to be a part of repository. Please remove that on the next commit